labextension/vpython/package.json

```
@@ -30,17 +30,18 @@
     "prepare": "npm run build"
   },
   "dependencies": {
-    "@jupyterlab/application": "^0.19.1",
-    "@jupyterlab/apputils": "^0.19.1",
-    "@jupyterlab/docregistry": "^0.19.1",
-    "@jupyterlab/notebook": "^0.19.1",
-    "@phosphor/disposable": "^1.1.2",
+    "@jupyterlab/application": "^1.1.2",
+    "@jupyterlab/apputils": "^1.1.2",
+    "@jupyterlab/docregistry": "^1.1.2",
+    "@jupyterlab/notebook": "^1.1.2",
+    "@phosphor/disposable": "^1.3.0",
     "script-loader": "^0.7.2"
   },
   "devDependencies": {
+    "@babel/types": "^7.6.1",
     "rimraf": "^2.6.1",
-    "script-loader": "^0.7.2",
-    "typescript": "~2.9.2"
+    "typescript": "^3.6.3",
+    "file-loader": "^4.2.0"
```

```suggestion
    "file-loader": "^4.2.0",
    "webpack": "^4.0.0"
```

Adding webpack eliminated this warning: "npm WARN file-loader@4.2.0 requires a peer of webpack@^4.0.0 but none is installed. You must install peer dependencies yourself."

---

Fine with me, Matt, but this is way outside my field of competence.

---

Hi Matt , I am OK with you merging this fix.

---

Incorporated this change in squash-commit f363918.